### ISSUE TITLE
Spinocerebellar ataxia 40

### ISSUE THREAD

**Issue by GARDInfoCenter** [obophenotype/human-phenotype-ontology#3401]
GARD received feedback from a person who states she was diagnosed with SCA40 at the age of 17 years and her child was recently diagnosed at the age of 14 years. HPO lists "adult onset" for SCA40. She would like to see this term dissociated from SCA40. Unfortunately, the person did not provide any genetic test results etc or detailed information about how she/her child were diagnosed. 

ATTN: @nicole.vasilevsky

Thanks, 
Maria Della Rocca
GARD

**Comment by drseb** [comment]
Please do not simply remove any annotations. It is clearly mentioned [in the clinical synopsis](https://www.omim.org/clinicalSynopsis/616053) and [in the full text](https://www.omim.org/entry/616053), where they write:

> Tsoi et al. (2014) reported a family from Hong Kong, China, in which 5 individuals had adult-onset spinocerebellar ataxia.

So either add another annotation with a different annotation and a different evidence code or leave it as it is, in case we are speaking about different SCA40 variants.

**Comment by pnrobinson** [comment]
I agree with Seb -- the annotations for the disease entities are the characteristic features. In the future, we can add individual observations like this -- there is always variability, and our downstream software is all designed to allow fuzziness in one way or the other.
